custom_components/bambu_lab/coordinator.py

```
@@ -177,6 +180,34 @@ def _service_call_send_gcode(self, event: Event):
         command['print']['param'] = f"{data.get('command')}\n"
         self.client.publish(command)
 
+    def _service_call_extrude_retract(self, event: Event):
+        data = event.data
+        if not self._service_call_is_for_me(data):
+            return
+
+        LOGGER.debug(f"_service_call_extrude_retract: {data}")
+
+        move = data.get('type').upper()
+        force = data.get('force')
+
+        if move not in ['EXTRUDE', 'RETRACT']:
+            LOGGER.debug(f"Invalid extrusion move '{move}'")
+            return False
+        
+        nozzle_temp = self.get_model().temperature.nozzle_temp
+        if force is not True and nozzle_temp < 170:
+            LOGGER.debug(f"Nozzle temperature too low to perform extrusion: {nozzle_temp}ºC")
```

Make this an error or maybe an info would be enough so there's a persistant explanation in the logs for why the command got dropped. We don't want incorrect usage to just be silently ignored.